packages/ibm-products-styles/src/components/Datagrid/styles/_draggableElement.scss

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright IBM Corp. 2022, 2022
+// Copyright IBM Corp. 2022, 2022, 2023, 2024
```

Comment:
The format/pattern for the years in the copyright headers is year file was created followed by year the file was last edited, so you only will have two years listed.

Reply:
Thanks @matthewgallo for the information. So I can update the year last edited as `2023`, right?
<img width="514" alt="Screenshot 2024-02-23 at 10 58 16 AM" src="https://github.com/carbon-design-system/ibm-products/assets/305492/a98fbf17-5d88-4bd6-ad98-a1848f51a9a1">

Reply:
Correct ^^

Reply:
@matthewgallo updated the copyright header `last modified year`.